dist/virtual-keyboard.directive.d.ts

```diff
@@ -9,6 +9,7 @@ export declare class NgVirtualKeyboardDirective {
     layout: KeyboardLayout | string;
     placeholder: string;
     type: string;
+    keyboardClose: EventEmitter<any>;
```

Comment:
Do we need that `<any>` here?

Reply:
Perhaps `void` is better here, EventEmitter type definition seems to require some type.

Reply:
yeah, `void` will tell that there isn't anything to emitted on that event - it's just an event, which makes totally sense in this context - imho